.github/workflows/ci_ut.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           submodule: true
       - name: Check License Header
-        uses: apache/skywalking-eyes@9bd5feb
+        uses: apache/skywalking-eyes@9bd5feb86b5817aa6072b008f9866a2c3bbc8587
         env:
```

Comment:
Should be consistent with the master, do not modify it

Reply:
'apache/skywalking-eyes' has been updated to the latest